include/experimental/debug.h

```diff
@@ -3,4 +3,39 @@
 
 #define BREAKPOINT __asm jr $ __endasm;
 
-#endif
\ No newline at end of file
+#include <string.h>
+#include <knightos/display.h>
+
+#define __DEBUG_TIMEOUT 100
```

Comment:
This should be an explicit function parameter, and not a define. e.g. `debug_hinted(screen, msg, hint, timeout);`

Reply:
Good point. I'll do that :)